src/components/Charts/LineChart.tsx

```diff
@@ -28,80 +39,105 @@ import LineChartOperations from '@/operations/charts/LineChartOperations';
+  }, [showBarChart, theme]);
 
   return (
     <>
-      <div className="w-full h-fit flex-col rounded-md bg-background">
-        <div
-          className={`w-full h-fit flex flex-row justify-between items-start gap-3 pl-3 pb-${HEADER_BOTTOM_PADDING}`}
-        >
-          <p className={`${TITLE_TEXT_SIZE} font-normal pt-2 flex flex-row items-center`}> {title} </p>
-          {fullScreenButton}
+      <div className={`w-full h-fit flex-col rounded-md ${transparentBackground ? 'bg-background' : 'bg-transparent'}`}>
+        <div className={`w-full h-fit flex flex-row justify-between items-start gap-1 pl-3 pb-${HEADER_PADDING}`}>
+          <h2 className={`${TITLE_TEXT_SIZE} font-normal pt-2 flex flex-row items-center`}> {title} </h2>
+          <div className="flex flex-row gap-0.5 pt-0.5 pr-0.5">
+            {
+              // button to hide/show the slider to manipulate the plotted x-axis range of the chart;
+              // can be disabled via `xAxisSlider`
+              xAxisSlider ? (
+                <Tooltip text="x-Axis Slider">
+                  <Button
+                    isIconOnly
+                    variant="light"
+                    size="sm"
+                    onPress={() => {
+                      setShowXAxisSlider(!showXAxisSlider);
+                    }}
+                  >
+                    <Settings className={`h-${ICON_BUTTON_SIZE} w-${ICON_BUTTON_SIZE}`} />
+                  </Button>
+                </Tooltip>
+              ) : null
+            }
+            {
+              // button to switch between line and bar chart; can be disabled via `barChartSwitch`
+              barChartSwitch ? (
+                <Tooltip text={`Switch to ${showBarChart ? 'Line' : 'Bar'} Chart`}>
+                  <Button isIconOnly variant="light" size="sm" onClick={() => setShowBarChart(!showBarChart)}>
+                    {showBarChart ? (
+                      <Diagram className={`h-${ICON_BUTTON_SIZE} w-${ICON_BUTTON_SIZE}`} />
+                    ) : (
+                      <Chart className={`h-${ICON_BUTTON_SIZE} w-${ICON_BUTTON_SIZE}`} />
+                    )}
+                  </Button>
+                </Tooltip>
+              ) : null
+            }
+            {
+              // button to trigger the full screen modal; rendered if `expandable`
+              expandable ? (
+                <Tooltip text="Enlarge Chart">
+                  <Button isIconOnly variant="light" size="sm" onClick={onOpen}>
+                    <Maximize4 className={`h-${ICON_BUTTON_SIZE} w-${ICON_BUTTON_SIZE}`} />
+                  </Button>
+                </Tooltip>
+              ) : null
+            }
+          </div>
         </div>
-        {descriptionText}
+        {
+          // description text element should only be rendered if description is available
+          description ? <p className={`w-full h-fit pb-4 ${DESCRIPTION_TEXT_SIZE} px-3`}> {description} </p> : null
```

Comment:
Use `description && <p className={w-full h-fit pb-4 ${DESCRIPTION_TEXT_SIZE} px-3}> {description} </p>` instead

Reply:
thanks for tip!